nixos/modules/programs/lazygit.nix

```diff
@@ -0,0 +1,35 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.lazygit;
+
+  settingsFormat = pkgs.formats.yaml { };
+in
+{
+  options.programs.lazygit = {
+    enable = lib.mkEnableOption (lib.mdDoc "simple terminal UI for git commands");
+
+    package = lib.mkPackageOption pkgs "lazygit" { };
+
+    settings = lib.mkOption {
+      inherit (settingsFormat) type;
+      default = { };
+      description = lib.mdDoc ''
+        Lazygit configuration.
+
```

Comment:
The documentation is too poor. See Home Manager's example: https://github.com/nix-community/home-manager/blob/a561ad6ab38578c812cc9af3b04f2cc60ebf48c9/modules/programs/lazygit.nix#L36

Reply:
I thought it's implementation details and it makes no sense to expose that to users.

Reply:
At the very least it's useful to link to something like https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md

The implementation details portion of it has still been useful sometimes, but it's the reference to documentation that saves the most time when consuming programs imo

Reply:
I already added the link to the doc. :)